docs/modules/grf.rst

```diff
@@ -0,0 +1,35 @@
+.. module:: glass.grf
+
+:mod:`glass.grf` --- Gaussian random fields
+===========================================
+
+.. currentmodule:: glass.grf
+
+
+Gaussian angular power spectra
+------------------------------
+
+.. autofunction:: compute
+.. autofunction:: solve
```

Comment:
As the `Parameters` section is missing, the docs have a bunch of vars without explanation of what they are

![image](https://github.com/user-attachments/assets/5b271fc2-391e-42a1-b0ef-19146bad0dbe)

Reply:
I added these. (I have been trying to add this comment a number of times now 😬)

Reply:
Great, thanks!